cmd/hypper/dependency_build.go

```diff
@@ -0,0 +1,15 @@
+package main
```

Comment:
cmd/hypper/dependency_build.go is missing a header.

Reply:
:+1: